__tests__/apiTests.test.js

```diff
@@ -88,4 +88,25 @@ describe("GET /api/articles/:article_id",()=>{
 
         })
     })
+})
+
+describe("GET /api/articles",()=>{
+    test("get status code 200, and all the articles in an array with comment_count and ordered by date descending",()=>{
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+            .then(({body})=>{
+                const articles=body.articles
+                expect(typeof articles).toBe("object")
+                expect(articles).toBeSortedBy("created_at",{descending: true})
```

Comment:
It would be preferable to have the sorting tested in its own test as we're checking something different to the actual structure of the response objects here